### ISSUE TITLE
Include CWE in the output

### ISSUE THREAD

**Issue by gbena** [nodesecurity/nsp#160]
It would be great if the output can include the CWE number of the vulnerability. 
This way, vulnerabilities can be grouped based on the CWE and thus easier to manage.

**Comment by evilpacket** [comment]
While it would be fantastic to have this data it would certainly be an effort to get it mapped. I'm going to map with a help wanted flag, I'm sure given a spreadsheet and some time and a few people we could bang this out, but it's not something we have direct time for at the moment.

**Comment by gbena** [comment]
Agreed that it would take a lot of effort to map all existing vulns in the advisory. We could probably start adding the CWE for new incoming vuls.

**Comment by evilpacket** [comment]
Our intern @bentleyjensen has been working on this when he has time. It's not a super high priority but wanted to communicate that some progress was being made.

**Comment by evilpacket** [comment]
As part of our backend data we now have CWE but not all the historical advisories have been filled in. I've also submitted an internal PR on our api to to get the cwe value into our output format and from there it's only cli formatters that need to care about it.
